### PR TITLE
Move layout and partial script to Shared dir

### PR DESCRIPTION
**Pull request by jbagga**
Addresses #340

### CODE REVIEW

File: korebuild-lock.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview2-15742
-commithash:21fbb0f2c3fe4a9216e2d59632b98cfd7d685962
+version:2.1.0-preview3-15744
```
Comment (pranavkm) [inline korebuild-lock.txt]: Did this update when you did `build -update`? It shouldn't be picking up `preview3`.

Comment (pranavkm) [reply korebuild-lock.txt]: Could you re-run this?

**Comment by javiercn** [comment]
We should also move _LoginPartial if its not already there.

**Comment by jbagga** [comment]
I removed the korebuild file @pranavkm 
@javiercn Added the template.json condition

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json [hunk 2]
```diff
@@ -60,6 +60,12 @@
             "Areas/**"
           ]
         },
+        {
+          "condition": "(IndividualB2CAuth)",
+          "rename": {
+            "Pages/_LoginPartial" : "Pages/Shared/_LoginPartial"
```
Comment (Eilon) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json]: File extensions aren't needed here?

Comment (jbagga) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json]: Not needed but I think nice to have. I'll add them

Comment (Eilon) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json]: Yeah it *looks* wrong, even if you assure me it's right 😄

**Comment by jbagga** [comment]
@Eilon Updated

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json [hunk 3]
```diff
@@ -60,6 +60,12 @@
             "Areas/**"
           ]
         },
+        {
+          "condition": "(IndividualB2CAuth)",
+          "rename": {
+            "Pages/_LoginPartial.cshtml" : "Pages/Shared/_LoginPartial.cshtml"
```
Comment (javiercn) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json]: I would have them living side by side in the `Pages/Shared` folder, name each of them explicitly `_LoginPartial.Identity.cshtml` and `_LoginPartial.OrgAuth.cshtml` and just make the rename be from `_LoginPartial.Identity.cshtml -> _LoginPartial.cshtml` and from `_LoginPartial.OrgAuth.cshtml -> _LoginPartial.cshtml`

The reason for it is that is not obvious what case each _LoginPartial file is for when you are editing them

Comment (Eilon) [reply src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json]: That sounds like a good idea to me.

File: src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json [hunk 4]
```diff
@@ -60,6 +60,12 @@
             "Areas/**"
           ]
         },
+        {
+          "condition": "(IndividualB2CAuth)",
```
Comment (javiercn) [inline src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/.template.config/template.json]: This condition is wrong, it should include Individual/MultiOrg

**Comment by jbagga** [comment]
I have modified existing conditions to rename and exclude appropriately @javiercn

**Comment by jbagga** [comment]
ping @pranavkm @javiercn Updated

**Pull request merged**